src/classes/issues-processor.ts

```diff
@@ -33,6 +33,13 @@ import {RateLimit} from './rate-limit';
 /***
  * Handle processing of issues for staleness/closure.
  */
+
+type state_reason_type =
+  | 'completed'
+  | 'reopened'
+  | 'not_planned'
+  | null
+  | undefined;
```

Comment:
What was the reason for adding this type?

Reply:
While I was running npm run test few test and test suites are failed for the code